merge_sort/merge_sort.js

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
move the code inside sorting/merge_sort. If the code already exists, no need to duplicate.